src/sentry/static/sentry/app/views/settings/components/teamSelect.jsx

```diff
@@ -36,40 +37,26 @@ class TeamSelect extends React.Component {
     confirmLastTeamRemoveMessage: PropTypes.string,
   };
 
-  state = {
-    teams: null,
-  };
-
-  componentDidMount() {
-    this.fetchTeams();
-  }
-
-  fetchTeams = debounce(query => {
-    const {organization} = this.props;
-    this.props.api
-      .requestPromise(`/organizations/${organization.slug}/teams/`, {
-        query: {query},
-      })
-      .then(teams => this.setState({teams}));
```

Comment:
This should have been doing an API call to get more teams as the typeahead was filled out, allowing the user to get outside the first 100 results.

Reply:
No, but the whole point is - at least as I understand it - that by using `withTeams` we always have access to _all_ of an org's teams, via the `teamStore`. So there's no issue with there only being 100 results, because that's never the case.

Reply:
For sure, that will re-introduce a dependency on the heavy org_details response which we are trying to wean ourselves off of so that we can elide teams/projects from it.